ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client/Jumping.hs

```diff
@@ -837,15 +865,21 @@ electNewDynamo ::
     LedgerSupportsProtocol blk
   ) =>
   Context m peer blk ->
-  STM m (Maybe (peer, ChainSyncClientHandle m blk))
+  STM m (TraceCsjReason -> TraceEventCsj peer blk, Maybe (peer, ChainSyncClientHandle m blk))
 electNewDynamo context = do
   peerStates <- cschcSeq (handlesCol context)
-  mDynamo <- findNonDisengaged peerStates
+  mDynamo <- do
+    -- prefer a 'Started' 'Objector', if any exists
```

Comment:
Note that `rotateDynamo` does **not** call `electNewDynamo`.

This is desirable, since the Objector isn't responsible for Dynamo rotations.

Reply:
I renamed `electNewDynamo` to `backfillDynamo` in order to emphasize this.